### PR TITLE
update draft name

### LINKED ISSUE #67: Change spec name to focus on Authz
Change name to focus on authorization (e.g. cross-domain authorization). Purpose of renaming is to create clarity that this is a mechanism for cross domain authz and that identity flows with the authorization. Add text to make it clear how identity chaining is achieved as a result of using this spec (the idea that the identity is following the authorization).

### PR DESCRIPTION
**Pull request by aaronpk**
closes #67

### CODE REVIEW

File: draft-ietf-oauth-identity-chaining.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ A home devices company provides a "Camera API" to enable access to home cameras.
 
 ## Overview
 
-The Identity Chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. The appendix include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
+The Identity and Authorization Chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. The appendix include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
```
Comment (bc-pi) [inline draft-ietf-oauth-identity-chaining.md:89]: we can probably do without the capitalization in regular text 
```suggestion
The identity and authorization chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. The appendix include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
```

Comment (bc-pi) [reply draft-ietf-oauth-identity-chaining.md:89]: I realize the capitalization predates your PR @aaronpk and you were just being consistent with the text immediately at hand. But your change made me notice it and seems like as good a time as any to fix it.

**Review by bc-pi — APPROVED** [review]
Looks good to me. Thanks for doing the work!

File: draft-ietf-oauth-identity-chaining.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ informative:
 
 --- abstract
 
-This specification defines a mechanism to preserve identity and call chain information across trust domains that use the OAuth 2.0 Framework.
+This specification defines a mechanism to preserve identity information and federate authorization across trust domains that use the OAuth 2.0 Framework.
 
 --- middle
```
Comment (arndt-s) [inline draft-ietf-oauth-identity-chaining.md:64]: I'm unfortunately not able to comment on the introduction section but we write there:
> Preserving this information is referred to as identity chaining. This document defines a mechanism for preserving identity chaining information across trust domains using a combination of ...

I believe with this change we should also change that section for example to
> This document referrers to this as "chaining" and defines a mechanism for preserving identity and authorization information across Domains.

**Review by arndt-s — APPROVED** [review]
LGTM, feel free to just close my comment and merge if you disagree.

File: draft-ietf-oauth-identity-chaining.md (hunk 2 above)
Comment (bc-pi) [reply draft-ietf-oauth-identity-chaining.md:64]: Yeah, agree we should also probably update that section somewhat.

Comment (bc-pi) [reply draft-ietf-oauth-identity-chaining.md:64]: 686f2d28f51470e86c8e7a0d90a989e6debd0b49 makes the update suggested by @arndt-s

**Comment by bc-pi** [comment]
Going to go ahead and merge this one as Pieter said we'd do today https://mailarchive.ietf.org/arch/msg/oauth/_nRDh3ciXDk1A9H_3CAs1DwPV5g/, which also says we'll publish a new draft but we can do that next week or so.

File: draft-ietf-oauth-identity-chaining.md [hunk 3]
```diff
@@ -58,26 +58,26 @@ informative:
...
 
 ## Requirements Language
 
 {::boilerplate bcp14-tagged}
 
-# Identity Chaining Across Trust Domains
+# Identity and Authorization Chaining Across Domains
```
Comment (kamronbatman) [inline draft-ietf-oauth-identity-chaining.md:72]: @aaronpk should this match the title field on line 11?

**Pull request merged**